app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -43,9 +44,42 @@ def test_field_returns_none_if_no_datetime_passed_in(self):
 
 class TestExperimentVariantSerializer(TestCase):
 
-    def test_serializer_outputs_expected_schema(self):
-        variant = ExperimentVariantFactory.create()
+    def test_serializer_outputs_expected_bool(self):
+        experiment = ExperimentFactory(pref_type=Experiment.PREF_TYPE_BOOL)
+        variant = ExperimentVariantFactory.create(experiment=experiment)
+        serializer = ExperimentRecipeVariantSerializer(variant)
+
+        self.assertEqual(type(serializer.data["value"]), bool)
+        self.assertEqual(
+            serializer.data,
+            {
+                "ratio": variant.ratio,
+                "slug": variant.slug,
+                "value": json.loads(variant.value),
```

Comment:
I would just hard code the values passed into the factory and hard them here in the check.